addons/movments_accounts_balances/controllers/controllers.py

```diff
@@ -79,3 +79,32 @@ def general_ledger_report(self, account_id, start_date, end_date):
             'ledger_data': ledger_data,
         }
 
+    @api.model
+    def create_bill(self, partner_id, invoice_date, due_date, reference, line_data, global_narration):
+        # Retrieve the currency ID for USD
+        currency_usd_id = self.env.ref('base.USD').id
+
+        Bill = self.env['account.move']
+        invoice_lines = []
+        for line in line_data:
+            invoice_line_vals = {
+                'name': line['description'],
+                'quantity': 1,
```

Comment:
In the previous code, the "quantity = 1 " was initially set by default, but I modified it as per your request.